.github/workflows/nightlyIntegrationTests.yml

```diff
@@ -7,16 +7,64 @@ on:
     - cron: 0 12 * * *
 
 jobs:
-  # We skip the LLVM pre-build-and-cache step since (presumably) it'll be in
-  # the cache already from the PR build.
+  # Build the LLVM submodule then cache it. Do not rebuild if hit in the
+  # cache.
+  build-llvm:
+    name: Build LLVM
+    runs-on: ubuntu-latest
```

Comment:
There may be a different between `ubuntu-latest` and the integration in the future. I'd suggest using the integration test image.

Reply:
I added the container line, same as the buildAndTest job.

Reply:
Sorry, didn't see that.

Reply:
It wasn't there before, I just meant I implemented your suggestion.  Sorry for the confusing way I said that, thanks for pointing it out!

Reply:
Thanks for the clarification. It would also help to see the individual commits (rather than amended & forced pushed) of your post-PR-submission revisions to better determine what you changed since last time someone reviewed it. There's no need for each PR to be one commit -- the squash merge will make it appear that way in the main branch.